### DISCUSSION TITLE
Linux - How exactly do i completely remove Brew from my Linux Mint system?

### DISCUSSION THREAD

**Discussion by presario42** [Homebrew/discussions#5277] [Linux]
### Additional Info

- [ ] I am using Windows Subsystem for Linux

### Output of `brew config`

```shell
Command "brew" not found.
```

### Output of `brew doctor`

```shell
Command "brew" not found.
```

### Description of issue

I have attempted to remove Brew (with certain success) but my system still screams stuff about brew being missing and stuff like that, is there a way i can 100% remove Brew from my system? This issue has broke some packages due to PATH issues and such and i want to solve those without the need of reinstalling Mint completely.

**Comment by SMillerDev** [comment]
Did you run the uninstall script? Because that is all that is needed to remove the brew install (although something manual might be required if the uninstaller tells you to)

**Reply by presario42** [reply]
Yep, that is what i did, but i believe there are remnants of Brew somewhere as opening the bash for example says that the linuxbrew install folder was not found

**Reply by SMillerDev** [reply]
Then you probably added something to `~/.bashrc` that references your brew install

**Reply by presario42** [reply]
Yep, there was a line in that file that Brew probably wrote, it's removed now

**Reply by ZhongRuoyu** [reply]
Homebrew never writes there. It's likely because you did this following the instructions printed by the installer at the end:

```
- Run these two commands in your terminal to add Homebrew to your PATH:
    (echo; echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"') >> /root/.bashrc
    eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
```

**Reply by lindhe** [reply]
> Did you run the uninstall script?

Where is that located?

**Reply by SMillerDev** [reply]
https://docs.brew.sh/FAQ#how-do-i-uninstall-homebrew
